netcode.c

```diff
@@ -8085,7 +8153,7 @@ void test_skillz_add_two_clients_to_match()
     {
         netcode_network_simulator_update( network_simulator, time );
 
-        for ( int j = 0; j < num_clients; ++j)
+        for ( j = 0; j < num_clients; ++j )
```

Comment:
Nit but it looks like you're out of the `for( i...` loop so I'd use `i` here.  Not even sure you need `j` defined?